pkg/component/labels/labels.go

```diff
@@ -10,6 +10,9 @@ const ComponentLabel = "app.kubernetes.io/component-name"
 // ComponentTypeLabel is kubernetes that identifies type of a component
 const ComponentTypeLabel = "app.kubernetes.io/component-type"
 
+// ComponentTypeLabel is kubernetes that identifies type of a component
```

Comment:
copy/paste?.  should be ComponentTypeVersion
Plus it looks like that comment doesn't make sense.